plugins/debridge-wali/src/index.ts

```diff
@@ -19,39 +18,43 @@ import { HttpUtils } from "./utils/http";
  * - Comprehensive error handling
  */
 export default createPlugin({
+  id: "@near-intents/debridge-data-provider",
+
   variables: z.object({
```

Comment:
It looks like you made the changes in your local repsoitroy, and then dragged and dropped them here for changes.

You must make a branch off of this repository, and only make a PR with the necessary changes.